Microwave.Test.Unit/UserInterfaceTest.cs

```diff
@@ -335,7 +343,21 @@ public void Cooking_CancelButton_LightCalled()
             light.Received(1).TurnOff();
         }
 
+        [Test]
```

Comment:
Maybe this test belongs in CookingController, but not sure. Shouldn't be referencing powerTube directly